erizoAPI/OneToManyProcessor.cc

```diff
@@ -84,16 +86,19 @@ NAN_MODULE_INIT(OneToManyProcessor::Init) {
 
 NAN_METHOD(OneToManyProcessor::New) {
   OneToManyProcessor* obj = new OneToManyProcessor();
-  obj->me = new erizo::OneToManyProcessor();
-  obj->msink = obj->me;
+  obj->me = std::make_shared<erizo::OneToManyProcessor>();
+  obj->msink = obj->me.get();
```

Comment:
we still use raw pointers through MediaSink and MediaSource, that's something we must change in a future PR I think

Reply:
Agreed